src/Illuminate/Database/Schema/Grammars/MySqlGrammar.php

```diff
@@ -560,6 +560,17 @@ protected function typeBinary(Fluent $column)
     }
 
     /**
+     * Create the column definition for a uuid type.
+     *
+     * @param  \Illuminate\Support\Fluent  $column
+     * @return string
+     */
+    protected function typeUuid(Fluent $column)
+    {
+        return 'char(36)';
```

Comment:
Storing uuid in `char(36)` leads to significant performance degradation.
Is it possible to tweak some stuff to use `binary(16)`